css/styles.css

```diff
@@ -484,7 +507,26 @@ textarea {
   background-image: url(../resources/image_dots.png);
   background-repeat: no-repeat;
   height: 175px;
-  visibility: hidden;
+  display: none;
+}
+
+.dottedDeco2 {
+  background-image: url(../resources/image_dots.png);
+  background-repeat: no-repeat;
+  height: 175px;
+  display: none;
+}
```

Comment:
Following the design template, kindly ensure that the text "Project" does not overlap with the background design as shown below:

Your design:
![image](https://user-images.githubusercontent.com/20554583/200152468-81d2ef93-29fa-40d4-896a-2ddcab37f9ab.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/200152479-98934a86-6eb3-41b9-9c92-0eb25336dc43.png)